lib/sort-lines.js

```diff
@@ -26,30 +26,30 @@ function sortLines(textEditor, startLine, endLine, algorithm, removeDuplicateVal
   });
 }
 
-function getUniqueArray(lines) {
+function getUniqueArray(lines, algorithm) {
   var unique = [];
   for (var i = 0; i < lines.length; ++i) {
-    if (unique.length === 0 || (unique[unique.length - 1] !== lines[i])) {
+    if (unique.length === 0 || (algorithm(unique[unique.length - 1], lines[i])) != 0) {
       unique.push(lines[i]);
     }
   }
   return unique;
 }
 
 function reverseCompare(a, b) {
-  return a < b ? 1 : -1;
+  return a = b ? 0 : a < b ? 1 : -1;
 }
 
 function caseInsensitiveCompare(a, b) {
-  return a.localeCompare(b, {sensitivity: 'base'});
+  return a.toLowerCase().localeCompare(b.toLowerCase(), {sensitivity: 'base'});
```

Comment:
Isn't `sensitivity: base` meant to do this based on the locale?

Reply:
It should do, but in my case (Win10x64, VSCode 1.20.1) it doesn't.

Reply:
@lynxnake what's your locale? Perhaps that's the reason it's not working? AFAIK toLowerCase is not how you're meant to do this from an i18n perspective.

Reply:
I have default locale:
```"locale":"en"```

Reply:
I agree that ```.toLowerCase()``` might be not the ideal solution, but in my case ```.localeCompare()``` was not working without it.

What can I do to have this pull request approved?

Reply:
@lynxnake does setting the second arg to `undefined` work? https://stackoverflow.com/a/49004987/1156119

Reply:
@Tyriar yes, it fixed this. Thanks